src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj

```diff
@@ -2,6 +2,8 @@
 
     <PropertyGroup>
         <TargetFrameworks>net6.0;$(NetAndroidTargetFramework);$(NetiOSTargetFramework)</TargetFrameworks>
+		<Description>ArcGIS Runtime controls and utilities for .NET MAUI apps including .NET for Android, .NET for iOS, .NET for macOS, or Windows UI 3 (WinUI 3)</Description>
```

Comment:
Should probably be .NET for MacCatalyst?

Reply:
@dotMorten I used based how it was [referred ](https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui#how-net-maui-works)in this microsoft doc. I can change if you think `.NET for MacCatalyst` is more correct